cypress/e2e/login.cy.js

```diff
@@ -1,11 +1,54 @@
-/// <reference types="cypress" />
+
 
 describe('Login page', () => {
   beforeEach(() => {
-    
+    cy.visit('login');
+  });
+
+  it('Login with valid creds (tomsmith/SuperSecretPassword!)', () => {
+    cy.get('[id="username"]')
```

Comment:
just to beautify
```suggestion
    cy.get('#username')
```

you can replace `id` with `#`